mods/ra/maps/evacuation/map.yaml

```diff
@@ -0,0 +1,1757 @@
+MapFormat: 11
+
+RequiresMod: ra
+
+Title: Evacuation
+
+Author: Scott_NZ
+
+Tileset: SNOW
+
+MapSize: 128,128
+
+Bounds: 16,16,96,96
+
+Visibility: MissionSelector, Lobby
```

Comment:
For me the way to solve https://github.com/OpenRA/OpenRA/issues/11692 is to
* add a multi-selection dropdown
* disable UI support for the setting
* make it a simple text edit widget

I wouldn't recommend just avoid using values and I don't see how having this on both the skirmish lobby and the mission selector contributes to negative UX. It just makes it easier to discover.